Add `adriancmiranda/glimpse.nvim` (#2326)
rockerBOO/awesome-neovim: README.md

- [Enigama/miss.nvim](https://github.com/Enigama/miss.nvim) - Simple popup with changed `unsaved` files, allowing you to save and open them. Helps to avoid forgetting to add something to GitHub or similar.
- [vodchella/hodur.nvim](https://github.com/vodchella/hodur.nvim) - Allows you to quickly open a file or copy URL located under cursor.
- [A7Lavinraj/fyler.nvim](https://github.com/A7Lavinraj/fyler.nvim) - File manager which can edit file system like a buffer with tree view.

> - [adriancmiranda/glimpse.nvim](https://github.com/adriancmiranda/glimpse.nvim) - Fast file previewer (images, videos, archives, fonts, SQLite) - inline via kitty Graphics or external pane, with Oil/Neo-tree/Telescope integration.

Reviewer: That loose `-` might confuse our checker script. Also a couple other suggestions with the wording.

```suggestion
- [adriancmiranda/glimpse.nvim](https://github.com/adriancmiranda/glimpse.nvim) - Fast file previewer (images, videos, archives, fonts, SQLite), inline via `kitty` Graphics or through an external pane, with `oil'nvim`, `neo-tree.nvim` and `telescope.nvim` integrations.
```

Author: Thanks, that makes sense.
The phrasing there is a bit too loose and could create false positives or parsing issues for checker scripts. I’ll revise it and improve the wording in the surrounding descriptions as well.
